### DISCUSSION TITLE
Strong markdown editor

### DISCUSSION THREAD

**Discussion by petrusmaximus1** [standardnotes/forum#2921] [General]
I believe that an application that wants to be based on standards should be based on the markdown format. Instead, the 'super' format is based on JSON.
It's a beautiful format, but if you try to decrypt a file and save it and read it with a basic text editor application, it comes out unreadable text (because of the JSON syntax) and that only makes sense if it is loaded in standard notes. Longevity is compromised in my opinion. If Standard notes were to fail (I hope not) one would first have to convert all notes from JSON to markdown or plain text. Longevity can only be provided by a format that is easily readable by a human (markdown, which was created on purpose). So there is no real standard, an independence from the app that only the markdown format could give.

it might be sufficient to have the possibility to export all notes in markdown and not just one and to have the possibility to decrypt backups directly in markdown. 

The markdown editors available in the app are separate pieces of software thrown in, not at all integrated with the rest of the software.

**Comment by affvalente** [comment]
I like the idea of exporting all to markdown
And decrypt from supernotes to markdown as well (from decrypt.html)
